tests/test_ops/test_inverted_residual_module.py

```diff
@@ -0,0 +1,40 @@
+import pytest
```

Comment:
Since InvertedResidual is moved to utils, the test case should also be moved accordingly.

Reply:
moved.